nemo/utils/exp_logging.py

```diff
@@ -123,12 +123,15 @@ def __init__(
         # Create work_dir if specified
         if work_dir:
             self.work_dir = work_dir
-            if add_time:
+            self.ckpt_dir = f'{self.work_dir}/checkpoints'
```

Comment:
Isn't it better to use os.path.joint instead of simple concatenation? It would avoid extra "/" in the final path.

Reply:
I agree, but it was like that in the original version

Reply:
We shouldn't use bad code as precedent to not make it better.
If both of you agree that something is better, please make the change.